plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-additional-information-block/attributes.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import formStepAttributes from '../../form-step/attributes';
+
+export default {
+	...formStepAttributes( {
+		defaultTitle: __( 'Additional order information', 'woocommerce' ),
+		defaultDescription: __(
+			'Enter any additional information about your order.',
+			'woocommerce'
+		),
+	} ),
+	className: {
+		type: 'string',
+		default: '',
+	},
+	lock: {
+		type: 'object',
+		default: {
+			move: true,
+			remove: true,
+		},
+	},
```

Comment:
DO we need it to be locked?

Reply:
No, we can unlock it if we want the merchant to be able to move it anywhere. I'm fine with doing that.